code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -8,7 +8,7 @@
 	desc = "An external organ that is too external."
 
 	///Unremovable is until the features are completely finished
```

Comment:
You forgot to change this comment, since they're no longer going to be unremovable.

Reply:
Woops, updated